### ISSUE TITLE
SQL error in installe

### ISSUE THREAD

**Issue by Hantree** [fisharebest/webtrees#2051]
I just downloaded the install zip from GitHub. During installation on the remote host server I encoutered this error: 
![schermafdruk van 2018-12-09 12-51-02](https://user-images.githubusercontent.com/45732857/49697161-424fa100-fbb4-11e8-92b3-aacc43898189.png)

**Comment by fisharebest** [comment]
This issue should have been fixed in https://github.com/fisharebest/webtrees/issues/2009

> I just downloaded the install zip from GitHub

Which version exactly?

Does your code contain this change?
https://github.com/fisharebest/webtrees/commit/aa5ee318333cef71c21536bb40713de6ce6b3177

If so, tell me about your MySQL server.  Version, config, etc.

**Comment by ungeahnt** [comment]
FYI: I've changed 2 weeks before manually the migration37.php (alpha-3) according aa5ee31 and the setup worked fine afterwards. But alpha-4 setup I can test only in the next few days.

**Comment by Hantree** [comment]
Ah, didn't expect such a quick answer. Thanlks!

Content understood: request  to change the ROW_FORMAT to DYNAMIC
is put forward to hosting party.

I used the download from GitHub which did not have a version number in
the package name.
Afterwards I downloaded 1.7.12 from the webtrees link and uploaded this
over the previous version.
Same error result during installation.

MyPHP reports version 50638

Could not find aa5ee31 because I do not know where to look for it.

Regards,
Hanri
